src/backup.js

```diff
@@ -188,7 +193,7 @@ const backup = async ()=>{
         if(collectionNames.length>0){
 
             const snapshotName          = `${new Date().getTime()}`;
-            const localBackupFileName   = `${snapshotName}.tar`;
+            const localBackupFileName   = `${snapshotName}.gz`;
```

Comment:
use .tgz extesion or .tar.gz

```suggestion
            const localBackupFileName   = `${snapshotName}.tar.gz`;
```